mimic/test/behavior_tests.py

```diff
@@ -0,0 +1,213 @@
+"""
+Module that contains utilities to automatically generate or help generate
```

Comment:
I think that "Module that contains utilities to" is a phrase that doesn't convey any additional meaning.  I think something like "Automatically generate tests for behaviors." would be better?